### PR TITLE
Add changelog entry for free license changes

### PR DESCRIPTION
**Pull request by pjlast**
As title says

## Test plan

Changelog entry

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ All notable changes to Sourcegraph are documented in this file.
 - Expiration of licenses is now handled differently. When a license is expired promotion to site-admin is disabled, license-specific features are disabled (exceptions being SSO & permission syncing), grace period has been replaced with a 7-day-before-expiration warning. [#47251](https://github.com/sourcegraph/sourcegraph/pull/47251)
 - Searcher will now timeout searches in 2 hours instead of 10 minutes. This timeout was raised for batch use cases (such as code insights) searching old revisions in very large repositories. This limit can be tuned with the environment variable `PROCESSING_TIMEOUT`. [#47469](https://github.com/sourcegraph/sourcegraph/pull/47469)
 - Zoekt now bypasses the regex engine for queries that are common in the context of search-based code intelligence, such as `\bLITERAL\b case:yes`. This can lead to a significant speed-up for "Find references" and "Find implementations" if precise code intelligence is not available. [zoekt#526](https://github.com/sourcegraph/zoekt/pull/526)
+- The Sourcegraph Free license has undergone a number of changes. Please contact support@sourcegraph.com if you have any questions or concerns.
```
Comment (kopancek) [inline CHANGELOG.md:42]: Do we have a blogpost or something that we could link to here?

**Review by kopancek — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (pjlast) [reply CHANGELOG.md:42]: Not yet that I'm aware of

Comment (sashaostrikov) [reply CHANGELOG.md:42]: Should we link a gh issue or a couple of PR links?

**Review by sashaostrikov — APPROVED** [review]
LGTM, added 1 comment

File: CHANGELOG.md (hunk 1 above)
Comment (pjlast) [reply CHANGELOG.md]: I added the main PR link 🤷

**Pull request merged**